src/_03_More_Array_Fun/MoreArrayFun.java

```diff
@@ -5,12 +5,66 @@
 public class MoreArrayFun {
 	//1. Create a main method to test the other methods you write.
 	
-	
+
 	
 	//2. Write a method that takes an array of Strings and prints all the Strings in the array.
 	
+
+	public static String printAll(String[] names) {
+		String listNames = "";
+		for(int i=0; i < names.length; i ++) {
+			if( i == names.length-1) {
+				listNames = listNames+ "" + names[i];
+			}else {
+				listNames = listNames+ "" + names[i]+ ",";
```

Comment:
I like the way you worked your if statement, I wouldn't have thought about that lol